bpf/http_trace.h

```diff
@@ -21,11 +21,12 @@
 #define REMOTE_ADDR_MAX_LEN 50 // We need 48: 39(ip v6 max) + 1(: separator) + 7(port length max value 65535) + 1(null terminator)
 #define HOST_LEN 256 // can be a fully qualified DNS name
 #define TRACEPARENT_LEN 55
+#define SQL_MAX_LEN 500
 
 // Trace of an HTTP call invocation. It is instantiated by the return uprobe and forwarded to the
 // user space through the events ringbuffer.
 typedef struct http_request_trace_t {
-    u8  type;
+    u8  type;                           // Must be first
```

Comment:
I guess we can't use C unions with the Bpf2go type converter

Reply:
I'm not sure, but can try it in another PR. Thanks for the review!